features/adresregels.feature

```diff
@@ -0,0 +1,133 @@
+    - Tussen postcode numeriek deel en postcode alfabetisch deel moet een spatie worden geplaatst
+    - Tussen postcode alfabetisch deel en woonplaatsnaam moeten twee spaties worden geplaatst
+
+    Scenario: er is geen huisnummer en geen huisletter
+        Gegeven adres met nummeraanduidingidentificatie "0957200000021241" heeft de volgende gegevens:
+        """
+        "straat": "Annelien Kappeyne van de Coppellostraat",
+        "huisnummer": 8,
+        "postcode": "6049HD",
+        "woonplaats": "Herten",
+        "korteNaam": "A K vd Coppellostr"
+        """
+        Als het adres wordt gevraagd met /adressen/0957200000021241
+        Dan bevat in het antwoord:
+        """
+        "adresregel1": "A K vd Coppellostr 8",
+        "adresregel2": "6049 HD  HERTEN"
+        """
+
+    Scenario: er is een huisletter en er is geen huinummertoevoeging
+        Gegeven adres met nummeraanduidingidentificatie "0344200000004821" heeft de volgende gegevens:
+        """
+        "straat": "Blauwkapelseweg",
+        "huisnummer": 1,
+        "huisletter": "A",
+        "postcode": "3572KA",
+        "woonplaats": "Utrecht",
+        "korteNaam": "Blauwkapelseweg",
+        """
+        Als het adres wordt gevraagd met /adressen/0344200000004821
+        Dan bevat in het antwoord:
+        """
+        "adresregel1": "Blauwkapelseweg 1 A",
+        "adresregel2": "3572 KA  UTRECHT"
+        """
+
+    Scenario: er is een huisnummertoevoeging die begint met een letter en er is geen huisletter
+        Gegeven adres met nummeraanduidingidentificatie "0417200000000354" heeft de volgende gegevens:
+        """
+        "straat": "Burgemeester van Nispen van Sevenaerstraat",
+        "huisnummer": 1,
+        "huisnummertoevoeging": "BEN",
+        "postcode": "1251KD",
+        "woonplaats": "Laren",
+        "korteNaam": "Burg v N v Sevenaerstr",
+        """
+        Als het adres wordt gevraagd met /adressen/0417200000000354
+        Dan bevat in het antwoord:
+        """
+        "adresregel1": "Burg v N v Sevenaerstr 1 BEN",
+        "adresregel2": "1251 KD  LAREN"
+        """
+
+    Scenario: er is een huisnummertoevoeging die begint met een cijfer en er is geen huisletter
+        Gegeven adres met nummeraanduidingidentificatie "0289200000017214" heeft de volgende gegevens:
+        """
+        "straat": "Bornsesteeg",
+        "huisnummer": 1,
+        "huisnummertoevoeging": "5A3",
+        "postcode": "6708GA",
+        "woonplaats": "Wageningen",
+        "korteNaam": "Bornsesteeg",
+        """
+        Als het adres wordt gevraagd met /adressen/0014200022188962
+        Dan bevat in het antwoord:
+        """
+        "adresregel1": "Bornsesteeg 1-5A3",
+        "adresregel2": "6708 GA  WAGENINGEN"
+        """
+
+    Scenario: er is een huisletter en een huisnummertoevoeging die begint met een letter
```

Comment:
Misschien goed om ook een voorbeeld op te nemen met huisnummer, huisletter en een huisnummertoevoeging die begint met een cijfer. Dan wordt het onderscheid mooi duidelijk tussen dat scenario en het scenario met alleen een huisnummer en huisnummertoevoeging die begint met een cijfer.

Reply:
@strijm bedoelde je deze: "Scenario: er is een huisletter en een huisnummertoevoeging die begint met een cijfer"?

Reply:
Ja